docs/docs/how-tos/using-hera.md

```diff
@@ -0,0 +1,341 @@
+
+w.create()
+```
+
+Parameters and environment variables will be created in the container using the names specified, e.g., `ENV_VAR1` and `arg`.  Environment variables can also be pulled in from other sources, such as a Docker registry.  See Hera's [Image Pull Secrets][image-pull-secrets] example for more.
+
+This script uses the `command` attribute to execute commands in the container.  The script above demonstrates how to access each type of variable (swap the commented commands to try each method).  If an executable, such as `echo` in the example above, is not specified, Hera defaults to `python`.  See the [description of `command` in the version 4 docs][command-description-v4-docs] for more on using the `command` attribute, and see the Kubernetes doc [Define a Command and Arguments for a Container][define-a-command-on-k8s] for more on executing commands in a pod.
+
+## Volume mounts
+
+Volume mounts can make it easy to pass environment variables, tokens, conda environments, and input data, as well as return output from the run directly to your home directory.  However, it might be more than you need, so consider simpler methods that only pass variables or that write output to the cloud.
+
+If you need to create an empty volume, you can specify whatever mount path you need, such as `/mnt/my_vol`.  Our example below serves two purposes.  If the volume did not already exist, it would create it, but because it is already specified for any linux container we spin up, we are effectively specifying the size that the volume should be when Docker creates it.  `/dev/shm` is typically small, like 64MB, but when running deep learning algorithms, we need a larger size, and anything from 2-16GB tends to work well for us.
+
+```python
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+from hera.shared import global_config
+from hera.workflows import Container, DAG, Volume, Workflow
+
+
+env_path = Path('.env').resolve()
+load_dotenv(env_path, verbose=True)
+global_config.token = os.environ['ARGO_TOKEN'].replace('Bearer ', '')
+global_config.host = os.environ['GLOBAL_CONFIG_HOST']
+global_config.namespace = os.environ['GLOBAL_CONFIG_NAMESPACE']
+
+
+with Workflow(generate_name="vol-", entrypoint="my-dag") as w:
+    container = Container(
+        name="container",
+        image="continuumio/miniconda3",
+        command=["df", "-h"],
+        volumes=[Volume(size="2Gi", mount_path="/dev/shm")]
+    )
+    with DAG(name="my-dag"):
+        A = container(name="A")
+
+w.create()
+```
+
+The next example adds a user's home directory as an existing volume mount.  As mentioned above, this can be a great way to quickly load environment variables from a `.env` file or pass datasets into your container.  Note that you should replace `<username>` with your Nebari username.
+
+TODO: is `jupyterhub-dev-share` the same for any user, or does this name change depending on the Nebari instance?
```

Comment:
@iameskild or @Adam-D-Lewis this TODO is a question for one of you.

Reply:
jupyterhub-dev-share is the same for all users